sites/partners/src/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -163,13 +163,22 @@ const BuildingDetails = ({
             validation={{ required: true }}
           />
         </GridCell>
-        <Field
-          label={t("t.neighborhood")}
-          name={"neighborhood"}
-          id={"neighborhood"}
-          placeholder={t("t.neighborhood")}
-          register={register}
-        />
+        <ViewItem label={t("t.neighborhood")} className={"mb-0"}>
+          <Select
+            id={`neighborhood`}
+            name={`neighborhood`}
```

Comment:
minor nit: I believe if you're only passing in strings for id and name can you just do:
```
id="neighborhood"
name="neighborhood"
```